dotnet/src/Extensions/PromptTemplate.Handlebars/Helpers/KernelHelperUtils.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+
+#pragma warning disable IDE0130 // Namespace does not match folder structure
+namespace Microsoft.SemanticKernel.PromptTemplate.Handlebars.Helpers;
+#pragma warning restore IDE0130
+
+/// <summary>
+/// Extension class to register additional helpers as Kernel System helpers.
+/// </summary>
+public static class KernelHelpersUtils
```

Comment:
This needs to be public? If yes, wee should come up with a different class name, but better to keep it internal if at all possible.